core/backend/src/TxnManager.ts

```diff
@@ -0,0 +1,208 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module iModels
+ */
+
+import { BeEvent, DbResult, Id64String, IModelStatus } from "@bentley/bentleyjs-core";
+import { ModelGeometryChangesProps } from "@bentley/imodeljs-common";
+import { BriefcaseDb, StandaloneDb } from "./IModelDb";
+import { IpcHost } from "./IpcHost";
+import { Relationship, RelationshipProps } from "./Relationship";
+
+/** @public */
+export enum TxnAction { None = 0, Commit = 1, Abandon = 2, Reverse = 3, Reinstate = 4, Merge = 5 }
+/** A string that identifies a Txn.

```

Comment:
Does the definition of "Txn" exist somewhere in the API docs? The string "Txn" isn't found in /docs/.

Reply:
see new file: `docs\learning\InteractiveEditing.md`. Obviously needs lots more (including explanation of `InteractiveEditingSession`).